lib/Dialect/AIEVec/Transforms/AIEVectorize.cpp

```diff
@@ -2410,8 +2397,9 @@ static void reassociateOpsInFunc(func::FuncOp func, VectState *state) {
   reassociateAddOpInFunc(func, state);
 }
 
-struct AIEVectorize : public AIEVectorizeBase<AIEVectorize> {
-  AIEVectorize() = default;
+struct AIEVectorize : public aievec::impl::AIEVectorizeBase<AIEVectorize> {
+  using Base::Base;
```

Comment:
Where do we use this?

Reply:
Line 2493:
`
pm.addPass(createAIEVectorize(options.getAIEVectorizeOptions()));
`
createAIEVectorize (autogenerated) eventually calls the inherited constructor in AIEVectorize.